packages/block-editor/src/components/block-card/index.js

```diff
@@ -103,6 +106,30 @@ function BlockCard( {
 		( { title, icon, description } = blockType );
 	}
 
+	const { isBlockHidden, hasHiddenParent } = useSelect(
+		( select ) => {
+			if ( ! clientId ) {
+				return { isBlockHidden: false, hasHiddenParent: false };
+			}
+			const { getBlockParents } = select( blockEditorStore );
+			const { isBlockHidden: _isBlockHidden } = unlock(
+				select( blockEditorStore )
+			);
```

Comment:
Note: You can use a single store getter for both private and public selectors.

Reply:
Thanks @Mamaduka! I'll update